packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -373,6 +400,48 @@ class AndroidWebViewController extends PlatformWebViewController {
       onShowFileSelector != null,
     );
   }
+
+  ///
+  /// Notify the host application that web content from the specified origin is attempting to use the Geolocation API,
```

Comment:
Remove the empty line above and separate the first sentence from the rest of the documentation: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph